src/dotnet/Azure.ClientSdk.Analyzers/TestReferenceWithInternalsVisibleTo/FriendAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+internal class FriendAttribute : Attribute
+{
+    public FriendAttribute(string friendAssembly)
```

Comment:
I take it we may consider validating `friendAssembly` in the future? If so, we may want to add a property. We can analyze it either way, but less code to check it's property later and code won't have to worry about whether it exists or not since it always will have existed.